### PR TITLE
Story2 see list of planets

### PR DESCRIPTION
**Pull request by cfbuckley14**

### CODE REVIEW

File: src/test/OrbitSvgTest.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        $actualOutput = OrbitSvg::generateOrbitSvg($orbitRadius, $planetPosition, $planetName, $planetRadius);
+        $this->assertEquals($expectedOutput, $actualOutput);
+    }
+
+    public function testFailureGenerateOrbitSvg()
+    {
+        $planetName = 'earth';
+        $planetRadius = '1000';
+        $planetPosition = '1000';
+        $orbitRadius = '50346345';
```
Comment (norayna) [inline src/test/OrbitSvgTest.php:28]: how come this is a failure case? because the numbers are too big?

File: src/test/OrbitSvgTest.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+use PlanetApp\OrbitSvg;
+
+class OrbitSvgTest extends TestCase
+{
+    public function testSuccessGenerateOrbitSvg()
+    {
+        $planetName = 'earth';
+        $planetRadius = '10';
+        $planetPosition = '100';
+        $orbitRadius = '50';
```
Comment (norayna) [inline src/test/OrbitSvgTest.php]: these numbers are in quotes for no reason

File: src/test/OrbitSvgTest.php (hunk 1 above)
Comment (norayna) [inline src/test/OrbitSvgTest.php]: quotes?

File: src/test/OrbitSvgTest.php [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<?php
```
Comment (norayna) [inline src/test/OrbitSvgTest.php]: remove this file lol

File: src/test/PlanetTest.php [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+class PlanetTest extends TestCase
+{
+    public function testSuccessCreateCardHtml()
+    {
+        $Planet = new Planet();
+        $Planet->setName('Uranus');
+        $Planet->setType('Ice giant');
+        $Planet->setImageSrc('pretend-url');
+
```
Comment (norayna) [inline src/test/PlanetTest.php]: variable name should be lowercase - $planet

File: src/Planet.php [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    public function setType(string $type): void
+    {
+        $this->type = $type;
+    }
+
+    public function createCardHtml(): string
+    {
+        return ('
+        <div class="planet-card">
+            <img src="images/' . $this->image_src . '">
```
Comment (norayna) [inline src/Planet.php]: needs alt attribute

File: style.css.map [hunk 6]
```diff
@@ -0,0 +1 @@
+{"version":3,"sourceRoot":"","sources":["style.scss"],"names":[],"mappings":"AAAA;EACE;EACA;EACA;EACA;;;AAGF;EACE;;;AAGF;EACE;EACA;;;AAGF;EACE;EACA;;;AAGF;EACE;EACA;EACA;EACA;EACA;;AAEA;EACE;EACA;;AAGF;EACE;EACA;EACA;;AAGF;EACE;IACE;;EAEF;IACE;;EAEF;IACE;;;;AAKN;EACE;EACA;EACA;EACA;EACA;EACA;;AAEA;EACE;;AAGF;EACE;EACA;EACA;;;AAIJ;EACE;IACE;IACA;IACA;IACA;IACA;;;EAEF;IACE;;;EAEF;IACE;;;EAEF;IACE;;;EAEF;IACE","file":"style.css"}
```
Comment (norayna) [inline style.css.map]: this file should not be in the repo

File: style.scss [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+      filter: drop-shadow(0 0 50px #c31b02);
+    }
+    100% {
+      filter: drop-shadow(0 0 50px #ef5f02);
+    }
+  }
+}
+
+.planet-card {
+  width: 90%;
```
Comment (norayna) [inline style.scss]: should be padding on the sides instead of width

File: index.php [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+<header>
+    <h1>Planet Plinky Plonk</h1>
+</header>
+<main>
+    <div class="title-image-container">
+        <img id="title-image" src="planet-plinky-plonk.png" alt="Planet Plinky Plonk"/>
+    </div>
+    <img class="sun-image mobile" src="sun.png" alt="Sun"/>
+
+    <h1>Planet Plinky Plonk</h1>
```
Comment (norayna) [inline index.php]: two h1s

File: index.php [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+    <link href="https://fonts.googleapis.com/css2?family=Archivo+Black&display=swap" rel="stylesheet">
+</head>
+<body>
+<header>
+    <h1>Planet Plinky Plonk</h1>
+</header>
+<main>
+    <div class="title-image-container">
+        <img id="title-image" src="planet-plinky-plonk.png" alt="Planet Plinky Plonk"/>
+    </div>
```
Comment (norayna) [inline index.php]: the h1 and the image should be in the same place ideally

**Review by norayna — CHANGES_REQUESTED** [review]
delete images.zip

File: src/test/OrbitSvgTest.php (hunk 2 above)
Comment (maxwell-01) [reply src/test/OrbitSvgTest.php]: file removed

File: src/test/OrbitSvgTest.php (hunk 1 above)
Comment (maxwell-01) [reply src/test/OrbitSvgTest.php]: file removed

File: src/test/OrbitSvgTest.php (hunk 3 above)
Comment (maxwell-01) [reply src/test/OrbitSvgTest.php]: file removed

File: src/test/PlanetTest.php (hunk 4 above)
Comment (maxwell-01) [reply src/test/PlanetTest.php]: done

File: src/Planet.php (hunk 5 above)
Comment (maxwell-01) [reply src/Planet.php]: done

File: style.css.map (hunk 6 above)
Comment (maxwell-01) [reply style.css.map]: removed

File: style.scss (hunk 7 above)
Comment (maxwell-01) [reply style.scss]: done

File: index.php (hunk 8 above)
Comment (maxwell-01) [reply index.php]: removed

File: index.php (hunk 9 above)
Comment (maxwell-01) [reply index.php]: done

**Comment by norayna** [comment]
on my massive monitor it doesn't look great 😳 the title is too big and the background loops

image

image

**Comment by norayna** [comment]
the planet type is too big compared to the planet name:
image

i would like it more like this:
image

**Comment by norayna** [comment]
on mobile, the planets are not making use of the space, i think the image should take up the full width
image

**Comment by norayna** [comment]
style.css needs to be removed from the repo and installation instructions need to be added to the readme

**Comment by norayna** [comment]
the logo image has uneven built in blank space on the left and right
image

**Review by norayna — APPROVED** [review]
approvedddddd

**Pull request merged**